benchmarks/trace/grpc/service.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package benchmarks;
+
+service Pinger {
+  rpc PingUniTraced(Ping) returns (Pong) {}
+  rpc PingUniUntraced(Ping) returns (Pong) {}
```

Comment:
You don't need two methods in the Pinger.

You need only a Ping and use the grpctrace package to enable tracing.